plugin/pom.xml

```diff
@@ -76,6 +76,16 @@
 
     <dependency>
       <groupId>io.fabric8</groupId>
+      <artifactId>fabric8-maven-enricher-karaf</artifactId>
```

Comment:
question is whether we maybe should have a single 'fabric8-maven-enricher-healthcheck' module containing both enrichers. They still can be switched on individually and since its part of the f-m-p plugin it doesnt make much sense to have separate modules so that only one might be included as a dependency. 

In general I avoid to introduce to many modules, because they typically increase the build time and release considerably and then there is a maintenance overhead. More specific, I'd try to avoid modules with a single class. Was there (in a 70 modules projects) and it was not much fun ;-)